sitemapindex.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<sitemapindex xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+xsi:schemaLocation="http://www.sitemaps.org/schemas/sitemap/0.9 http://www.sitemaps.org/schemas/sitemap/0.9/siteindex.xsd"
+xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+  <sitemap>
+    <loc>https://www.mdanalysis.org/docs/sitemap.xml</loc>
+  </sitemap>
```

Comment:
I don't think you have the dev API docs in here.

```suggestion
  </sitemap>
  <sitemap>
    <loc>https://docs.mdanalysis.org/1.0.0/sitemap.xml</loc>
  </sitemap>
  <sitemap>
    <loc>https://docs.mdanalysis.org/2.0.0-dev0/sitemap.xml</loc>
  </sitemap>
```

Reply:
On purpose – I think it will be confusing to get two hits for the same question (on our search), especially for normal users who never install develop. 

Google will index the dev docs anyway and that's fine but I think we should make an effort to direct users to the docs for the released version.

Reply:
Sounds good. I have been meaning to ask (maybe here is not the best place) if you are keen to keep the manually-built www.mdanalysis.org/docs repo+url, or mainly use docs.mdanalysis.org/ for both the release and development docs when the Travis changes are backported to 1.x? If docs.mdanalysis.org is to mainly be the development docs, then that URL should redirect to 2.0.0-dev0, instead of 1.0.0 as it does now.

Reply:
I am not keen on manually-anything, the more we automate the better. 

The https://www.mdanalysis.org/docs is used widely so I think the best thing would be to have it redirect to the latest stable docs. Is this something we can do?

Reply:
For right now I will use https://docs.mdanalysis.org/1.0.0/sitemap.xml and we must update eventually but it would be good to have a stable location such as https://docs.mdanalysis.org/sitemap.xml